README.md

```diff
@@ -25,93 +21,108 @@ Never heard of Interledger before, or you would like to learn more? Here are som
 
 Please read the [contribution guidelines](.github/contributing.md) before submitting contributions. All contributions must adhere to our [code of conduct](.github/CODE_OF_CONDUCT.md).
 
-## Local Development Environment
+## Project Structure
 
-### Prerequisites
+This is a monorepo containing several packages:
 
-- [Docker](https://docs.docker.com/get-docker/)
-- [NVM](https://github.com/nvm-sh/nvm)
+- **`api/`** - Hono-based API server running on Cloudflare Workers. Used by tools embedded on websites to fetch their config, handle payments, and manage probabilistic revenue sharing.
+- **`frontend/`** - Remix-based React frontend application. Provides the configuration interface where publishers customize their Web Monetization tools (banners, widgets, link tags).
+- **`components/`** - Lit-based web components for publishers. Contains reusable web components that get embedded into publisher websites.
+- **`cdn/`** - Content delivery network package. Delivers the embeddable scripts and their related assets that publishers include on their websites to show monetization tools.
+- **`shared/`** - Shared utilities and types
+- **`localenv/`** - Local development environment setup. Provides local S3 simulation for testing configuration storage during development.
 
-### Environment Setup
+## Prerequisites
 
-```sh
-# Install Node 20
-nvm install lts/iron
-nvm use lts/iron
+- [Node.js](https://nodejs.org/) 20+ (LTS recommended)
+- [pnpm](https://pnpm.io/) 9.15.9+
 
-# Install pnpm using Corepack
-corepack enable
-```
+## Installation
 
-If you do not have `corepack` installed locally you can use `npm` or `yarn` to install `pnpm`:
+1. **Install Node.js**:
 
-```sh
-npm install pnpm -g
-# or
-yarn install pnpm -g
-```
+   ```sh
+   # Use nvm
+   nvm install 24
+   nvm use 24
+   ```
```

Comment:
I was looking into nvm-windows supporting `.nvmrc`, and they don't plan to support it any time soon. There are some ways to do so in git-bash and Powershell, but Command Prompt is less ergonomic.

I created [this gist](https://gist.github.com/sidvishnoi/f795887659f5bec32f01a7ec9e788fc1), which maybe helpful to link here, and not require us to hardcode Node.js version (24) in README.